backend/src/test/kotlin/org/loculus/backend/controller/submission/SubmitEndpointTest.kt

```diff
@@ -178,9 +178,7 @@ class SubmitEndpointTest(
             organism = OTHER_ORGANISM,
             groupId = groupId,
         )
-            .andExpect(status().isOk)
-            .andExpect(content().contentType(APPLICATION_JSON_VALUE))
-            .andExpect(jsonPath("\$.length()").value(1))
+            .andExpect(status().isBadRequest)
```

Comment:
Should we expect a more specific message to check we're in the right code path?

Reply:
good idea, I was being lazy there 😬

Reply:
done: https://github.com/loculus-project/loculus/pull/3769/commits/0c92b8a6c4201f3869b7e3f0b2fd9f28c050e547